static/js/components/TopAppBar_test.js

```diff
@@ -33,37 +33,73 @@ describe("TopAppBar component", () => {
 
     it("has a button to collapse the menu", () => {
       assert.isOk(
-        shallow(<TopAppBar currentUser={user} />)
+        shallow(<TopAppBar currentUser={user} location={null} />)
           .find("button")
           .exists()
       )
     })
+
+    it("does not have a login/register on ecommerce bulk page", () => {
+      const location = { pathname: "/ecommerce/bulk/", hash: "", search: "" }
+      const wrapper = shallow(
+        <TopAppBar currentUser={user} location={location} />
+      )
+      assert.isNotOk(wrapper.find("UserMenu").exists())
+      assert.isNotOk(wrapper.find("MixedLink").exists())
+    })
+
+    it("does not have a login/register on ecommerce bulk receipt page", () => {
+      const location = { pathname: "/ecommerce/bulk/", hash: "", search: "" }
```

Comment:
what is the difference b/w these two tests here in terms of location ?

Reply:
Ah, missed updating the path to `/ecommerce/bulk/receipt/`